product list/product.css

```diff
@@ -0,0 +1,79 @@
+.product-container{
+    display: flex;
+    margin: 1rem;
+}
+.filter-container{
+    position: fixed;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: .5rem;
```

Comment:
It will be better to write decimal numbers with a zero, makes it easier to read.

Reply:
Okay will do the changes!!